packages/server/src/script/resendSignupEmails.ts

```diff
@@ -34,22 +36,45 @@ type Where = [
  */
 export const makeResendEmailsConditions = (reason: string, ...where: Where[]): ResendEmailsConditionData => {
```

Comment:
Would you mind updating the sample conditions.ts to include an 'in' with more than 10 conditions? Actually, if you could add support for reading a CSV file of emails to conditions.ts, that would be super helpful.